boxsdk/pagination/page.py

```diff
@@ -61,7 +61,7 @@ def __getitem__(self, key):
         item_class = self._translator.translate(item_json['type'])
         item = item_class(
             session=self._session,
-            object_id=item_json['id'],
+            object_id=item_json['id'] if 'id' in item_json else None,
```

Comment:
I think this is what's going to be necessary - solves the `**kwargs` in RecentItem as well.